src/app/header/show-meet-details/show-meet-details.component.ts

```diff
@@ -57,4 +68,34 @@ export class ShowMeetDetailsComponent implements OnInit {
         }
       })
   }
+
+  setMeetToDelete(item){
+    this.MeetToDelete=item;
+    this.deletedMeetEnabled=true;
+   
```

Comment:
What is the use of this function?

Reply:
This function is for further scheduling techniques in schedule meet